### ISSUE TITLE
[pyinstaller] Cannot read properties of undefined (reading 'api')

### ISSUE THREAD

**Issue by Liripo** [r0x0r/pywebview#1504]
### Specification
- pywebview version: 5.3
- operating system: windows
- web renderer: default

### Description

The js api works when I run it directly from source, but it doesn't work after compiling with pyinstaller.

example code:
```py
import webview
from pathlib import Path
class Api:
    def add(self,a,b):
        return a+b


webview.create_window('Hello world', url= str(Path('web/index.html').absolute()),js_api=Api())
webview.start(http_server=True,debug=True)
```

web/index.html
```html

<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>My First App</title>
</head>
<body>
  <h1>JS to Python</h1>
  <main>
      <button onclick="onClickOk()">OK</button>
  </main>
  
  <script>
      async function onClickOk(){
          let res = await window.pywebview.api.add(10, 15);
          console.log(res);
      }
  </script>
</body>
</html>
```

![image](https://github.com/user-attachments/assets/c7e2427a-3cc1-4526-8b26-5bdcb6560292)

**Comment by Barafu** [comment]
I came to report the same thing. In PyInstaller, `pywebview` is unavailable, and `window.pywebviewready` event never happens. Win 11, python 3.12.7.  Running directly as `python main.py` everything works.

```
pip freeze
altgraph==0.17.4
bottle==0.13.2
cffi==1.17.1
clr-loader==0.2.6
packaging==24.1
pefile==2023.2.7
proxy_tools==0.1.0
pycparser==2.22
pyinstaller==6.11.0
pyinstaller-hooks-contrib==2024.9
pythonnet==3.0.4
pywebview==5.3
pywin32-ctypes==0.2.3
setuptools==75.2.0
typing_extensions==4.12.2
```

**Comment by SirCypkowskyy** [comment]
I also have the same problem and would love to find a solution to it

**Comment by r0x0r** [comment]
Due to thhe JS files refactor, pyinstaller hook needs to be updated. I pushed an updated hook to the master along with the pyproject.toml configuration, which should take care of this problem. I will release a hotfix version shortly.

**Comment by Barafu** [comment]
Cool! And I chose that very day to start learning PyWebView and webdev in general.

**Comment by r0x0r** [comment]
Fixed in 5.3.1

**Comment by aylitat** [comment]
I still hava this problem.
version 5.3.2
I use nuitka + pywebview before,and it works well.
but it doesnt work now.

**Comment by aylitat** [comment]
> 5.3.1

maybe the fix on 5.3.1 makes some problem to nuitka?
